quickstarts/hello-rules/hellorules-client/pom.xml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
not needed line.

Reply:
Did you forget to remove this line?

Reply:
Did you forget to remove this line?

Reply:
Did you forget to remove this line?

Reply:
What line? I'm not seeing a new line in the pom files. Also the hellorules pom.xml already had the new line after the end tag. We need this new line or not?

Reply:
@ricardozanini , the newline shows up here: https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/8538fee84be2d1dad78b6f56751d53e3b850515d/quickstarts/hello-rules/hellorules-client/pom.xml#L1

Reply:
@errantepiphany thanks! updated! sorry guys, my vim went crazy yesterday with some plugins I was playing around.